### ISSUE TITLE
How to get equation(7) in the paper

### ISSUE THREAD

**Issue by wisdomGEsLA** [junkangwu/Adap_tau#2]
Sorry to be a bother! I think it's excellent work but I'm confused about how to get equation(7) in the paper, it appears that I cannot obtain this result if I adhere to the derivative calculation method of softmax cross-entropy loss

**Comment by junkangwu** [comment]
Apologies for the delayed response. 

In regards to the derivation of formula 7, upon further investigation, we discovered a minor issue in that we derived it based on the softmax loss without log() outside. Empirical evidence suggests that using ada_tau leads to smoother training. After all, the value inside the logarithm approaches 1/2, which helps prevent problems with exploding or vanishing gradients.

Regarding your reminder, we will make the necessary corrections in the upcoming version on arXiv. Thank you for your inquiry!
